### ISSUE TITLE
MagnitudeToDecibel Incorrect Scaling Constant

### ISSUE THREAD

**Issue by daniel-deychakiwsky** [keunwoochoi/kapre#143]
Its my understanding that when converting to decibel, the scaling constant should be 20.0 for magnitude mel/specgrams and 10.0 for power mel/specgrams. It seems the Kapre [implementation](https://github.com/keunwoochoi/kapre/blob/master/kapre/backend.py#L70) anticipates a magnitude mel/specgram but scales it as if it is a power mel/sepcgram. Notice that Librosa's implementation, the `power_to_db` function, applies a 10.0 scaling factor but [anticipates](https://github.com/librosa/librosa/blob/222b2287e69bcdf2f99000a0d93cbd25af7a325b/librosa/core/spectrum.py#L1753) a power mel/specgram. Of course, this is a difference of a scaling but thought I'd mention it here. Any thoughts on that?

**Comment by raraz15** [comment]
I agree with this observation. Another potential issue arises when someone applies dynamic range compression later, assuming the features are in dB when they're actually not. This mismatch could lead to unintended processing artifacts or incorrect results.
